frontend/src/api/apollo-client.ts

```diff
@@ -22,7 +22,7 @@ const headerLink = setContext((_, { headers }) => {
 });
 
 const httpLink = new HttpLink({
-  uri: JSON.parse(process.env.NEXT_PUBLIC_DEV) ? `http://localhost:8000/graphql/` : `https://api.test-pledge4future.heigit.org/graphql/`
+  uri: JSON.parse(process.env.NEXT_PUBLIC_DEV ? process.env.NEXT_PUBLIC_DEV  as string : 'false' ) ? `http://localhost:8000/graphql/` : `https://api.test-pledge4future.heigit.org/graphql/`
```

Comment:
Would this also work when we deploy on the production instance at pledge4future.org?

Reply:
Hm, good point. But I think this change should not alter the way the setup works right now. So I suggest we create a new, general concept to distinguish between local test setup and deployed setup. I will create an issue for that :)